packages/menus/src/containers/MenuContainer.js

```diff
@@ -509,6 +508,7 @@ class MenuContainer extends ControlledComponent {
       id,
       defaultFocusedIndex
     } = this.getControlledState();
+    const defaultNode = getDocument(this.props).body;
```

Comment:
Could we just assign `getDocument(this.props).body` as the default value for `appendToNode`?

Reply:
Didn't think that was possible since `defaultProps` are static?

Reply:
Ah I didn't mean defaultProps but a default value to the destructure value like we do with the `render` prop e.g.

```js
 render() {
    const {
      children,
      render = children,
      trigger,
      eventsEnabled,
      popperModifiers,
      appendToNode = getDocument(this.props).body,
      zIndex
    } = this.props;
    //...

    return createPortal(menu, appendToNode);
  }
```